mh/server.py

```diff
@@ -206,19 +207,24 @@ def fileno(self):
         """
         return self.socket.fileno()
 
-    def serve_forever(self):
+    def serve_forever(self, broadcast_queue):
+        self.broadcast_queue = broadcast_queue
```

Comment:
Passing the `broadcast_queue` as a parameter seems overkill. Can't it be created in the server's constructor method?